app/locale/da.coffee

```diff
@@ -136,12 +136,12 @@ module.exports = nativeDescription: "dansk", englishDescription: "Danish", trans
     request_demo_subtitle: "Anmod om en demo og få dine elever i gang på mindre end en time."
     get_started_title: "Opsæt din klasse i dag"
     get_started_subtitle: "Opsæt en klasse, tilføj dine elever, og overvåg deres fremgang når de lærer datalogi."
-    request_demo: "Anmod demo"
-#    request_quote: "Request a Quote"
+    request_demo: "Anmod om en demo"
+    request_quote: "Anmod om et tilbud"
     setup_a_class: "Opsæt klasse"
     have_an_account: "Har du allerede en konto?"
     logged_in_as: "Du er i øjeblikket logget ind som"
-    computer_science: "Datalogikurser for alle aldre" # {change}
+    computer_science: "Vores kurser dækker fra grundlæggende syntax til avancerede begreber" # {change} missing self-paced
```

Comment:
I know what self-paced means.
I just don't know how to express it well in danish.